{"name": "test_courier_login_invalid_login_code_404", "status": "passed", "start": 1739103059329, "stop": 1739103059385, "uuid": "b74deae6-cadb-466b-ac3b-5686f9a93f09", "historyId": "3198a6e214d93d4a9402b5c55e9c7839", "testCaseId": "3198a6e214d93d4a9402b5c55e9c7839", "fullName": "tests.test_courier_login.TestCourierLogin#test_courier_login_invalid_login_code_404", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_courier_login"}, {"name": "subSuite", "value": "TestCourierLogin"}, {"name": "host", "value": "DESKTOP-4NMQM7J"}, {"name": "thread", "value": "6928-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_courier_login"}]}